### PR TITLE
Feature/hrt386 case activityreport

### PR DESCRIPTION
**Pull request by jasattilia**
## Link to JIRA ticket

Add a link to the JIRA ticket that the changes in this PR describe.

### *What is the problem we're trying to solve*

Add a summary of the problem and any additional context that helps explain what the issue we're trying to solve is.

### *What changes have we introduced*

Add descriptions of what changes were made to address the problem and provide a solution.

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: HousingRegisterApi/V1/Gateways/Interfaces/IActivityHistory.cs [hunk 1]
```diff
@@ -11,5 +15,12 @@ public interface IActivityHistory
         /// <param name="application"></param>
         /// <param name="activity"></param>
         void LogActivity(Application application, EntityActivity<ApplicationActivityType> activity);
+
+        /// <summary>
+        ///  Gets the activity history for an application
+        /// </summary>
+        /// <param name="applicationId"></param>
+        /// <returns></returns>
+        Task<IList<ActivityHistoryResponseObject>> GetActivites(Guid applicationId);
```
Comment (tcmorris) [inline HousingRegisterApi/V1/Gateways/Interfaces/IActivityHistory.cs:24]: Spelling mistake.

File: HousingRegisterApi/V1/UseCase/GetInternalReportUseCase.cs [hunk 2]
```diff
@@ -18,17 +18,20 @@ public class GetInternalReportUseCase : IGetInternalReportUseCase
         private readonly ILogger<GetInternalReportUseCase> _logger;
         private readonly IFileGateway _fileGateway;
         private readonly IApplicationApiGateway _gateway;
+        private readonly IActivityHistory _activityGateway;
```
Comment (tcmorris) [inline HousingRegisterApi/V1/UseCase/GetInternalReportUseCase.cs:21]: `IActivityHistoryGateway`

**Review by tcmorris — APPROVED** [review]

File: HousingRegisterApi/V1/Gateways/Interfaces/IActivityHistory.cs (hunk 1 above)
Comment (jasattilia) [reply HousingRegisterApi/V1/Gateways/Interfaces/IActivityHistory.cs]: done

File: HousingRegisterApi/V1/UseCase/GetInternalReportUseCase.cs (hunk 2 above)
Comment (jasattilia) [reply HousingRegisterApi/V1/UseCase/GetInternalReportUseCase.cs]: i changed it to activitygateway cos its shorter

**Pull request merged**